playwright-e2e/utils/test-utils.ts

```diff
@@ -242,3 +243,21 @@ export function studyShortName(study: StudyEnum): {shortName: string | null; rea
   }
   return {shortName, realm};
 }
+
+export function shuffle(array: any[]): any[] {
+  for (let i = array.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [array[i], array[j]] = [array[j], array[i]];
+  }
+  return array;
+}
+
+export async function toHaveScreenshot(page: Page, locator: Locator | string, name: string): Promise<void> {
+  // https://github.com/microsoft/playwright/issues/18827
```

Comment:
good comment to add (thumbs up - written since I can't find emoji for this textbox)